app/models/cherrypick_task.rb

```diff
@@ -21,9 +21,26 @@ class CherrypickTask < Task # rubocop:todo Metrics/ClassLength
   #
   # @return [CherrypickTask::ControlLocator] A generator of control locations
   #
-  def new_control_locator(batch_id, total_wells, num_control_wells, wells_to_leave_free: DEFAULT_WELLS_TO_LEAVE_FREE)
-    CherrypickTask::ControlLocator.new(batch_id:, total_wells:, num_control_wells:, wells_to_leave_free:)
+
+  # rubocop:disable Metrics/ParameterLists
```

Comment:
Do we understand why this method is needed?  It seems like the only benefit it's offering is the default value for `wells_to_leave_free`.  We can probably avoid having all these parameters on the method by accepting a single `arguments` parameter instead and then passing the values from that through to `CherrypickTask::ControlLocator.new` after doing `parameters[:wells_to_leave_free] ||= DEFAULT_WELLS_TO_LEAVE_FREE`.  It's always better to try to comply with the Rubocop rules rather than disable them, unless we agree that rules should be turned off.